lib/phlex/rails/rendering.rb

```diff
@@ -0,0 +1,11 @@
+module Phlex
+  module Rails
+    module Rendering
+      def render(...)
+        _raw ApplicationController.render(...)
```

Comment:
I expect we can use [`ActionController::Rendering#render_to_string`](https://api.rubyonrails.org/classes/AbstractController/Rendering.html#method-i-render_to_string) for this.